math/utils_test.go

```diff
@@ -494,3 +494,33 @@ func TestGetSortedElementsByDecWeightDesc(t *testing.T) {
 	keys := alloraMath.GetSortedElementsByDecWeightDesc(m)
 	require.Equal(t, expected, keys)
 }
+
+func TestGetQuantileOfDecs(t *testing.T) {
+	m := []alloraMath.Dec{
```

Comment:
add tests where: 
* empty set
* 1 element set
* negative errors 
* inclusion of NaN/ negative errors in the set. 

These can be generated using AI in a table-based manner.

Reply:
this was never done @kpeluso (i think)

Reply:
I think it is done actually. It doesn't show as "outdated" here because those particular lines are unchanged, but below these lines you see all the extra cases.

Reply:
Preemptively resolving but feel free to pop open if issue

Reply:
oh I see, not really labeled in the comment. If it's cool I'm going to push a commit

Reply:
ok I didn't see that

I think while we're trying to write quality code, we can do better than this

1. I would like to see a comment on top of each `data` field dictating what this test case is (empty set, 1 element set, etc etc) - I didn't even realize we had covered the test cases because they blended in with the code. So just adding a comment above each table element labeling which test case it is (alternatively can put the name first so it's the first thing the eye is drawn to, and improve what the name field actually says, because several of these are non-descriptive names "mixed data"- mixed data of what? mixed negative and positive data)
2. The infinity case that diego listed is not covered
3. I believe we need to add a `t.Parallel()` somewhere in here
4. Why is `TestGetQuantileOfDecsWithInvalidQuantile` a separate test, can we just combine it...?